doc/releases/changelog-0.39.0.md

```diff
@@ -4,220 +4,384 @@
+  [(#6308)](https://github.com/PennyLaneAI/pennylane/pull/6308)
+
+* The `SampleMP.process_samples` method has been updated to support using JAX tracers for samples, allowing 
+  compatiblity with qjit workflows.
+  [(#6211)](https://github.com/PennyLaneAI/pennylane/pull/6211)
+
 <h4>Capturing and representing hybrid programs</h4>
 
-* `qml.wires.Wires` now accepts JAX arrays as input. Furthermore, a `FutureWarning` is no longer raised in `JAX 0.4.30+`
-  when providing JAX tracers as input to `qml.wires.Wires`.
+* `qml.wires.Wires` now accepts JAX arrays as input. Furthermore, a `FutureWarning` is no longer raised 
+  in `JAX 0.4.30+` when providing JAX tracers as input to `qml.wires.Wires`.
   [(#6312)](https://github.com/PennyLaneAI/pennylane/pull/6312)
 
-* Differentiation of hybrid programs via `qml.grad` and `qml.jacobian` can now be captured
-  into plxpr. When evaluating a captured `qml.grad` (`qml.jacobian`) instruction, it will
-  dispatch to `jax.grad` (`jax.jacobian`), which differs from the Autograd implementation
-  without capture. Pytree inputs and outputs are supported.
+* A new function called `qml.capture.make_plxpr` has been added to take a function and create a `Callable` 
+  that, when called, will return a PLxPR representation of the input function.
+  [(#6326)](https://github.com/PennyLaneAI/pennylane/pull/6326)
+
+* Differentiation of hybrid programs via `qml.grad` and `qml.jacobian` can now be captured with PLxPR. 
+  When evaluating a captured `qml.grad` (`qml.jacobian`) instruction, it will dispatch to `jax.grad` 
+  (`jax.jacobian`), which differs from the Autograd implementation without capture. Pytree inputs and 
+  outputs are supported.
   [(#6120)](https://github.com/PennyLaneAI/pennylane/pull/6120)
   [(#6127)](https://github.com/PennyLaneAI/pennylane/pull/6127)
   [(#6134)](https://github.com/PennyLaneAI/pennylane/pull/6134)
 
-* Improve unit testing for capturing of nested control flows.
+* Unit testing for capturing nested control flow has been improved.
   [(#6111)](https://github.com/PennyLaneAI/pennylane/pull/6111)
 
-* Some custom primitives for the capture project can now be imported via
-  `from pennylane.capture.primitives import *`.
+* Some custom primitives for the capture project can now be imported via `from pennylane.capture.primitives import *`.
   [(#6129)](https://github.com/PennyLaneAI/pennylane/pull/6129)
 
-* All higher order primitives now use `jax.core.Jaxpr` as metadata instead of sometimes
-  using `jax.core.ClosedJaxpr` and sometimes using `jax.core.Jaxpr`.
+* All higher order primitives now use `jax.core.Jaxpr` as metadata instead of sometimes using `jax.core.ClosedJaxpr` 
+  or `jax.core.Jaxpr`.
   [(#6319)](https://github.com/PennyLaneAI/pennylane/pull/6319)
 
-* `FermiWord` class now has a method to apply anti-commutator relations.
-   [(#6196)](https://github.com/PennyLaneAI/pennylane/pull/6196)
+<h4>Improvements to fermionic operators</h4>
```

Comment:
I think slightly similar feedback to what I mentioned above; it's not quite clear to me as a reader _what the improvement is_ here for me. What can I do now that I couldn't do before?